sql/create_users_and_user_roles.sql

```diff
@@ -0,0 +1,92 @@
+SET @nursesalt = (SHA2(SUBSTRING(MD5(RAND()), -10), 512));
+SET @doctorsalt = (SHA2(SUBSTRING(MD5(RAND()), -10), 512));
+SET @receptionsalt = (SHA2(SUBSTRING(MD5(RAND()), -10), 512));
+INSERT IGNORE INTO person (uuid,gender,date_created,creator) VALUES ('3f64faba-ad31-11e9-a2a3-2a2ae2dbcce4','F',NOW(),('4'));
+INSERT IGNORE INTO person_name (person_id,uuid,given_name,family_name) 
+VALUES
+((SELECT person_id FROM person WHERE uuid = '3f64faba-ad31-11e9-a2a3-2a2ae2dbcce4'),(uuid()),('Nurse'),(''));
+INSERT IGNORE INTO person (uuid,gender,date_created,creator) VALUES ('3f64fd1c-ad31-11e9-a2a3-2a2ae2dbcce4','F',NOW(),('4'));
+INSERT IGNORE INTO person_name (person_id,uuid,given_name,family_name) 
+VALUES
+((SELECT person_id FROM person WHERE uuid = '3f64fd1c-ad31-11e9-a2a3-2a2ae2dbcce4'),(uuid()),('Doctor'),(''));
+INSERT IGNORE INTO person (uuid,gender,date_created,creator) VALUES ('3f64fe70-ad31-11e9-a2a3-2a2ae2dbcce4','F',NOW(),('4'));
+INSERT IGNORE INTO person_name (person_id,uuid,given_name,family_name) 
+VALUES
+((SELECT person_id FROM person WHERE uuid = '3f64fe70-ad31-11e9-a2a3-2a2ae2dbcce4'),(uuid()),('Reception'),(''));
+INSERT IGNORE INTO users (system_id,username,password,salt,date_created,uuid,person_id,creator)
+VALUES 
+('Nurse', 'Nurse', SHA2(CONCAT('Nurse#123',@nursesalt), 512), @nursesalt, NOW(), '3bfbd6ec-ad1c-11e9-a2a3-2a2ae2dbcce4',(SELECT person_id FROM person WHERE uuid = '3f64faba-ad31-11e9-a2a3-2a2ae2dbcce4'),('4')),
+('Doctor', 'Doctor', SHA2(CONCAT('Doctor#123',@doctorsalt), 512), @doctorsalt, NOW(), '3bfbdbba-ad1c-11e9-a2a3-2a2ae2dbcce4',(SELECT person_id FROM person WHERE uuid = '3f64fd1c-ad31-11e9-a2a3-2a2ae2dbcce4'),('4')),
+('Reception', 'Reception', SHA2(CONCAT('Reception#123',@receptionsalt), 512), @receptionsalt, NOW(), '3bfbdd40-ad1c-11e9-a2a3-2a2ae2dbcce4',(SELECT person_id FROM person WHERE uuid = '3f64fe70-ad31-11e9-a2a3-2a2ae2dbcce4'),('4'));
+INSERT IGNORE INTO provider 
+(person_id,creator,date_created,uuid)
+VALUES
+((SELECT person_id FROM person WHERE uuid = '3f64faba-ad31-11e9-a2a3-2a2ae2dbcce4'),'4',NOW(),'1ad6aa26-adfa-11e9-a2a3-2a2ae2dbcce4'),
+((SELECT person_id FROM person WHERE uuid = '3f64fd1c-ad31-11e9-a2a3-2a2ae2dbcce4'),'4',NOW(),'1ad6ac88-adfa-11e9-a2a3-2a2ae2dbcce4'),
+((SELECT person_id FROM person WHERE uuid = '3f64fe70-ad31-11e9-a2a3-2a2ae2dbcce4'),'4',NOW(),'1ad6adf0-adfa-11e9-a2a3-2a2ae2dbcce4');
+INSERT IGNORE INTO user_role (user_id,role)
+VALUES 
+((SELECT user_id FROM users WHERE username = 'Nurse'),
```

Comment:
@prashantsable1991 please create a variable that stores the result of `SELECT user_id FROM users WHERE username = 'Nurse'` (this will prevent repeating the queries). This can be done for Doctor and Reception as well